### PR TITLE
Modal link documentation

### LINKED ISSUE #2054: Modal template documentation
Please provide a`dosomething_helpers` README.md to document how the modal themeable function / template works.

### PR DESCRIPTION
**Pull request by blisteringherb**
Fixes #2054

### CODE REVIEW

File: lib/modules/dosomething/dosomething_helpers/README.md [hunk 1]
```diff
@@ -26,3 +26,17 @@ display all nodes that reference them.
...
+
+## Modal Theme Function
+
+This module contains a theme function, `theme_modal_links()` and a template, modal-links.tpl.php, for putting modal links and their associated script tags on a page. The function currently supports 'faq', 'more_facts', and 'partner_info'.  These items should be passed the theme function in a $modals array:
+````
+theme('modal_links', array('modals' =>
+    array(
+      'faq' => $vars['faq'],
+      'partner_info' => $vars['partner_info'],
+      'more_facts' => $
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_helpers/README.md]: should say `$vars['more-facts']`

**Comment by barryclark** [comment]
@aaronschachter

**Comment by angaither** [comment]
:+1:

File: lib/modules/dosomething/dosomething_helpers/README.md [hunk 2]
```diff
@@ -26,3 +26,18 @@ display all nodes that reference them.
 
 This module is responsible for adding all of those character counters in the
 node edit forms.
+
+## Modal Theme Function
+
+This module contains a theme function, `theme_modal_links()` and a template, modal-links.tpl.php, for putting modal links and their associated script tags on a page. The function currently supports 'faq', 'more_facts', and 'partner_info'.  The $issue variable should be passed in if using 'more_facts'. These items should be passed the theme function in a $modals array:
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_helpers/README.md:32]: This sentence is a bit confusing: `The $issue variable should be passed in if using 'more_facts'`.

I think what it means is that the `$issue` variable is the link that will open a `more_facts` modal?

**Pull request merged**